app/post_metadata.go

```diff
@@ -490,6 +491,12 @@ func (a *App) getLinkMetadata(requestURL string, timestamp int64, isNewPost bool
 			return nil, nil, nil, appErr
 		}
 
+		if referencedPost == nil {
```

Comment:
should we have similar checks for `referencedChannel`, etc.?

Reply:
@catalintomai If this is nil, we can't go any further, so there is no other data to check. I added this anyway in case that's also an issue. I also added the post id in the log for context.

Reply:
@mickmister , yes, that is under the assumption that they are all (`referencedPost`, etc) nil or not-nil in sync, which I don't know if it holds true.